src/Form/Admin.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Drupal\islandora_audio\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Link;
+use Drupal\Core\Url;
+
+/**
+ * Module settings form.
+ */
+class Admin extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'islandora_audio_admin';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
+    $config = $this->config('islandora_audio.settings');
+
+    $config->set('islandora_lame_url', $form_state->getValue('islandora_lame_url'));
+    $config->set('islandora_audio_defer_derivatives_on_ingest', $form_state->getValue('islandora_audio_defer_derivatives_on_ingest'));
+    $config->set('islandora_audio_vbr_quality', $form_state->getValue('islandora_audio_vbr_quality'));
+    $config->set('islandora_audio_obj_fallback', $form_state->getValue('islandora_audio_obj_fallback'));
+
+    islandora_set_viewer_info('islandora_audio_viewers', $form_state->getValue('islandora_audio_viewers'));
+
+    $config->save();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return ['islandora_audio.settings'];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $form_state->loadInclude('islandora', 'inc', 'includes/utilities');
+    $form_state->loadInclude('islandora_audio', 'inc', 'includes/admin.form');
+    $config = $this->config('islandora_audio.settings');
+    $lame = $config->get('islandora_lame_url');
```

Comment:
See https://github.com/discoverygarden/islandora_book/pull/1/files#diff-d8f967483d111635d7104a486841cd7cR48 for a translation of that annon func that was in use.